### PR TITLE
DM-6999: Logging framework migration

### PR DESCRIPTION
**Pull request by hchiang2**

### CODE REVIEW

File: tests/openFiles.py [hunk 1]
```diff
@@ -87,7 +87,7 @@ def getAstrom(self):
 
         conf = measAstrom.ANetBasicAstrometryConfig()
         return measAstrom.ANetBasicAstrometryTask(config=conf, andConfig=andconfig,)
-        #logLevel=pexLog.Log.DEBUG)
+        #logLevel=pexLog.Log.TRACE)
```
Comment (isullivan) [inline tests/openFiles.py]: I suggest you just delete this dead code, rather than updating it.

File: tests/testJoinMatchListWithCatalog.py [hunk 2]
```diff
@@ -54,14 +54,14 @@ def setUp(self):
         mi.assign(smallExposure.getMaskedImage(), smallExposure.getBBox())
 
         logLevel = Log.INFO
-        #logLevel = Log.DEBUG
+        #logLevel = Log.TRACE
```
Comment (isullivan) [inline tests/testJoinMatchListWithCatalog.py]: As above, I suggest you just delete this commented-out code.

**Comment by isullivan** [comment]
I had only very minor comments on the code. I don't have the new lsst.log so the changed tests all fail for me, but assuming that they do all pass with the new framework this branch can be merged.

File: tests/openFiles.py (hunk 1 above)
Comment (hchiang2) [inline tests/openFiles.py]: It's not removed.

File: tests/testJoinMatchListWithCatalog.py (hunk 2 above)
Comment (hchiang2) [inline tests/testJoinMatchListWithCatalog.py]: I'm leaving this one un-touched.

**Pull request merged**